components/CancelButton/CancelButton.styled.js

```diff
@@ -0,0 +1,14 @@
+import { styled } from "styled-components";
+import Link from "next/link";
+
+export const StyledCancelButton = styled(Link)`
+  background-color: red;
+  writing-mode: horizontal-tb !important;
+  margin: 0em;
+  padding-inline: 6px;
```

Comment:
In this case, you can also use the following to achieve the same outcome:
```suggestion
  padding: 0 6px;
```